client/state/selectors/is-requesting-reader-teams.js

```diff
@@ -0,0 +1,3 @@
+export default function isRequestingReaderTeams( state ) {
+	return !! state.reader.teams.isRequesting;
```

Comment:
a perfect bit of logic for the middleware

there is no meaning of "requesting terms" when in a web socket context

also "requesting terms" is zapped on browser refreshes and doesn't matter to the app for rendering its state

should be good enough to store as a boolean in middleware local state and the real kicker is that it's the middleware's responsibility to poll, not the components'. only want to poll if the reader is open? we can watch for `NEED_READER_TEAMS` messages and only poll if we've seen one (from a render of the reader) within the past N seconds